src/generatePOT/generatePOT.js

```diff
@@ -13,29 +13,57 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- * Copyright (c) 2020 (original work) Open Assessment Technologies SA
+ * Copyright (c) 2020-2022 (original work) Open Assessment Technologies SA
  *
  */
 
 /**
  * Writes the strings to POT file
  *
- * Skips the duplicate strings
+ * Removes the duplicate strings
  * @param {Map<Object, Object[]}
  * @returns {string}
  */
 module.exports = function generatePOT(strings) {
     let potContent = '';
 
+    const msgids = new Set(); // avoids duplicates
+    const msgid_plurals = new Set();
+
+    const getPotHeader = () => `msgid ""
+msgstr ""
+"Project-Id-Version: \\n"
+"POT-Creation-Date: ${new Date().toISOString().slice(0,16).replace('T',' ')}\\n"
+"PO-Revision-Date: \\n"
+"Last-Translator: \\n"
+"Language-Team: \\n"
+"Language: en\\n"
+"MIME-Version: 1.0\\n"
+"Content-Type: text/plain; charset=UTF-8\\n"
+"Content-Transfer-Encoding: 8bit\\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\\n"
```

Comment:
Did you try if the `Language` or `Plural-Forms` are going to update the PO file values? I mean, if you have a PO file with `de` in the  `Language` header, when you update the PO file with the POT, does it keep the `de` value or does it update it with the POT header?

Reply:
I've checked it in POEdit, and the values in the language-specific PO header won't be overwritten by those in the POT header by any updates after the initial PO creation.